core/tests/protractor/collections.js

```diff
@@ -82,7 +110,93 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('checks for console errors in a collection with one node', function() {
+    users.login('player@collections.com');
+    browser.get('/search/find?q=');
+    browser.waitForAngular();
+    element.all(by.css(
```

Comment:
Maybe this should be something like library.clickSummaryTileWithTitle('Test Collection 2')?

Reply:
Actually, this part is not needed if I am finding collection ID in beforeeach() itself, will remove this.

Reply:
Removed it